test/conformance/walk.go

```diff
@@ -85,12 +87,17 @@ func (v *visitor) convertToConformanceData(at *ast.BasicLit) {
 	cd.Description = ""
 	for _, line := range lines {
 		line = strings.TrimSpace(line)
-		if strings.HasPrefix(line, "Testname:") {
+		if ok, _ := regexp.MatchString("^\\s*Testname\\s*:", line); ok {
 			line = strings.TrimSpace(line[9:])
```

Comment:
If we allowing variable amounts of whitespace(\\s*) then 9 is no longer a reliable amount to trim by.
You could use the position of the first : (+1) for the start index

Reply:
That is a nice point.
Actually if contianing spaces like "Testname : test1", the testname becomes ": test1" which is different from expected.
I guess we can change here like
```
if sline := regexp.MustCompile("Testname\\s*:\\s*").Split(line, -1); len(sline) == 2 {
    line = sline[1]
    ...
```

Reply:
Thanks for catching this bug. I fixed it as per the suggestion and works well.